third_party/java_grpc_library.bzl

```diff
@@ -0,0 +1,133 @@
+def _path_ignoring_repository(f):
```

Comment:
Where did this code come from?

Reply:
@oferb vendored-in [`java_grpc_library.bzl`](https://github.com/grpc/grpc-java/blob/master/java_grpc_library.bzl) with slight changes. Shall we provide a link?